packages/fxa-admin-server/src/test/lib/resolvers/account-resolver.spec.ts

```diff
@@ -29,32 +30,45 @@ const EMAIL_BOUNCE_1 = randomEmailBounce(USER_1.email);
 const USER_2 = randomAccount();
 
 describe('accountResolver', () => {
+  // Used to get the shape of the return type
```

Comment:
Comment/code is a little unclear to me (probably since not very familiar with typescript), but I take it this tells compiler that `contextDummy` is a boolean value?

Reply:
Yea, found a better solution though.